pkg/bootkube/status.go

```diff
@@ -25,7 +25,7 @@ const (
 	doesNotExist = "DoesNotExist"
 )
 
-func WaitUntilPodsRunning(pods []string, timeout time.Duration, selfHostedEtcd bool) error {
+func WaitUntilPodsRunning(pods []string, timeout time.Duration, selfHostedEtcd bool, etcdServiceIP string) error {
```

Comment:
@dghubble Can you add a todo to clean this up. we should not pass in the serviceIP for a func that waiting for pod running.

@hongchaodeng I think we should move etcdMigrate code out of this function. We can start migration after the wait func reties in the future?